bindings/zeebe/command/cancel_instance.go

```diff
@@ -14,10 +14,10 @@ import (
 	"github.com/dapr/components-contrib/bindings"
 )
 
-var ErrMissingWorkflowInstanceKey = errors.New("workflowInstanceKey is a required attribute")
+var ErrMissingProcessInstanceKey = errors.New("processInstanceKey is a required attribute")
 
 type cancelInstancePayload struct {
-	WorkflowInstanceKey *int64 `json:"workflowInstanceKey"`
+	ProcessInstanceKey *int64 `json:"processInstanceKey"`
```

Comment:
Please, keep the old name too so it is not a breaking change.

Reply:
Looking at our documentation, this component is not in GA. So, it can be a breaking change. This change is OK.

https://docs.dapr.io/operations/components/certification-lifecycle/#alpha
https://docs.dapr.io/operations/components/certification-lifecycle/#beta
https://docs.dapr.io/reference/components-reference/supported-bindings/#zeebe-camunda